book/src/api/node_delete.md

```diff
@@ -14,13 +14,9 @@ mutation {
                 status: "INACTIVE"
             },
             $DELETE: {
-                force: true
```

Comment:
@maldrake : was this an error in the docs, or did the force flag get removed?

Reply:
I removed the force flag.  In Cypher, the default behavior is to return an error if a query tries to DELETE nodes that still have relationships. The force flag mapped to a DETACH DELETE in Cypher, which forces the deletion of both a node and associated relationships.

Gremlin's behavior is a little different.  Deleting a node automatically deletes the associated relationships, similar to the behavior of DETACH DELETE.  Simulating the behavior of a Cypher DELETE (without DETACH) would require an extra round trip to the database, with additional logic to test for the presence of any relationships before deleting a node.

I decided that it in this case, it was cleaner to support the lowest common denominator of the two databases' behavior, rather than adding a potentially hidden cost to the force flag. If a client wants to check for relationships before deleting a node, it can still do so. It would just need to be done as a separate query.